src/main/java/io/vlingo/common/completes/AndThen.java

```diff
@@ -0,0 +1,39 @@
+package io.vlingo.common.completes;
+
+import java.util.function.Function;
+
+public class AndThen<I, O, NO> implements Operation<I, O, NO> {
```

Comment:
As a general comment is there any better naming for the generics?

Reply:
I => Input
O => Output
NO => NextOutput

I think they could be better, I will change them 😄 .